pcweb/pages/docs/gallery.py

```diff
@@ -531,6 +531,8 @@ def gallery() -> rx.Component:
         ),
         max_width="80em",
         margin_x="auto",
-        margin_top="80px",
+        margin_top="120px",
         height="100%",
     )
+
+# fixme
```

Comment:
Why is the extra comment here? Is there a todo for later

Reply:
Just deleted it, I forget to delete it